docs/pages/build-reference/troubleshooting.mdx

```diff
@@ -85,13 +85,9 @@ It's not yet possible to increase memory limits on your build workers, [only one
 
 ## Verify that your JavaScript bundles locally
 
-When you get the `Metro encountered an error` during a native iOS build, it means Metro bundler failed to bundle the app. This unfortunately runs at the end of the native build process, meaning it takes a while to fail.
+When a build fails with `Metro encountered an error` (iOS) or `Task :app:bundleReleaseJsAndAssets FAILED` (Android), it means Metro bundler failed to bundle the app's JavaScript code while trying to embed it in your app's binary. This error message is usually followed by a syntax error or other details related to why bundling failed. A standard React Native project is configured to perform this step late in the Xcode / Gradle build step, meaning it can take a while to see this error.
```

Comment:
```suggestion
When a build fails with `Task :app:bundleReleaseJsAndAssets FAILED` (Android) or `Metro encountered an error` (iOS), it means Metro bundler was unable to bundle the app's JavaScript code while trying to embed it in your app's binary. This error message is usually followed by a syntax error or other details about why bundling failed. Unfortunately, a standard React Native project is configured to perform this step late in the Gradle/Xcode build step, meaning it can take a while to see this error.
```

A suggestion to follow our [writing style guide for referencing Android and iOS](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web) and updated the verbiage.